mr/src/main/java/org/apache/mahout/cf/taste/impl/similarity/precompute/MultithreadedBatchItemSimilarities.java

```diff
@@ -26,7 +26,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 
-import com.google.common.collect.Lists;
+import java.util.ArrayList;
 import com.google.common.io.Closeables;
```

Comment:
Argh, I blindly replaced only collections! I could have removed Closeables as well, would be trivial. I'm assuming you will be doing it, based on your previous comment.